media/js/base/core-datalayer.js

```diff
@@ -40,11 +40,7 @@ if (typeof window.Mozilla.Analytics === 'undefined') {
         if (!isModernBrowser) {
             return 'false';
         }
-        return document.querySelector('video') !== null ||
-            document.querySelector('iframe[src^="https://www.youtube"]') !==
-                null
-            ? 'true'
-            : 'false';
+        return document.querySelector('video') !== null ? 'true' : 'false';
```

Comment:
Yeah, yo be honest I was more tempted to just remove all this GA stuff since: a) no one ever looks at or uses this data to my knowledge, and b) we're also not tracking it in GA4. But then I didn't want to make this PR even bigger, so I left it and maybe we can remove it all later?